### DISCUSSION TITLE
[QMP VI] AccuWeather API faltante

### DISCUSSION THREAD

**Discussion by foviedo** [dds-jv/foro#141] [Consultas de objetos] [answered]
Hola! Un detalle:
En el enunciado dice que actualizaron AccuWeather para que entienda el mensaje getAlertas, pero esto no está subido
Igual bien podría hacer un método dummy que no devuelva nada, pero lo comento por las dudas

**Comment by roli93** [comment, marked as answer]
No lo subimos adrede. Recuerden que no les pedimos q el código compile. Por ende no deberían tener problemas en usar el método en pseudocódigo sin tener su implementación ;)

Saludos!
